pytorch_lightning/plugins/precision/cpu_native_amp.py

```diff
@@ -0,0 +1,46 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Union
+
+import torch
+
+from pytorch_lightning.plugins.precision.native_amp import NativeMixedPrecisionPlugin
+from pytorch_lightning.utilities import _TORCH_GREATER_EQUAL_1_10
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+
+
+class CPUNativeMixedPrecisionPlugin(NativeMixedPrecisionPlugin):
+    """
+    Plugin for CPU native mixed precision training with :mod:`torch.cpu.amp`.
+
+    Args:
+        precision: Whether to use torch.float16 (16) or torch.bfloat16 (bf16).
+            Currently only torch.bfloat16 is supported.
```

Comment:
Is it worth mentioning 16 if it's not supported? Is it going to be supported before the 1.10 release?

Reply:
good point, don't think so! removed it